src/wolfio.c

```diff
@@ -1078,6 +1078,13 @@ int wolfIO_HttpProcessResponse(int sfd, const char** appStrList,
                 start[len] = 0;
             }
             else {
+            #ifdef WOLFSSL_NONBLOCK_OCSP
```

Comment:
Does requiring the user to turn on nonblocking support make sense?  That is, isn't that usually the default in wolfSSL.